src/index.jsx

```diff
@@ -53,6 +53,8 @@ initialize({
         ENTERPRISE_SUBSIDY_BASE_URL: process.env.ENTERPRISE_SUBSIDY_BASE_URL || null,
         EXPERIMENT_2_ID: process.env.EXPERIMENT_2_ID || null,
         EXPERIMENT_2_VARIANT_2_ID: process.env.EXPERIMENT_2_VARIANT_2_ID || null,
+        EXPERIMENT_3_ID: process.env.EXPERIMENT_3_ID || null,
```

Comment:
I know it doesn't fit the previous format, but do we want to call it something better than just experiment 3?

Reply:
This is a good idea -- I updated it to something more comprehensible `PREQUERY_SEARCH_EXPERIMENT_ID`